src/views/RecIM/views/Home/index.jsx

```diff
@@ -333,15 +334,38 @@ const Home = () => {
     </Card>
   );
 
-  if (!profile) {
-    return loading ? <GordonLoader /> : <GordonUnauthenticated feature={'the Rec-IM page'} />;
-  } else {
-    return (
-      <>
-        <Header>{headerContents}</Header>
-        {loading ? (
-          <GordonLoader />
-        ) : (
+  let headerAlert = (
+    /*participant?.Status === 'Pending' &&*/ <Alert
```

Comment:
Looks like you commented this out for testing? This should be restored before we merge.

Reply:
Yea noticed that :\